### PR TITLE
Fixed typos in some cells

### PR DESCRIPTION
**Pull request by aliciacanipe**
@eteq here are the typo fixes. I'm hoping this time there won't be conflicts? I think the issue was because I made my fork when "test_env" was the default branch, so my default fork is also "test_env" and when I went to synch my fork I grabbed old changes. Hopefully this is a cleaner PR, but let me know if I did it wrong.

### CODE REVIEW

File: pipeline_products_session/jwst-data-products-part2-live.ipynb [hunk 1]
```diff
@@ -482,7 +482,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Load the WFSS image into the appropriate model (hint: ImageModel)\n"
+    "# Load the WFSS image (wfss_rate_file[1]) into the appropriate model (hint: ImageModel)\n"
```
Comment (eteq) [inline pipeline_products_session/jwst-data-products-part2-live.ipynb:485]: Maybe this to make it clear it's code:
```suggestion
    "# Load the WFSS image (`wfss_rate_file[1]`) into the appropriate model (hint: ImageModel)\n"
```

File: pipeline_products_session/jwst-data-products-part2-live.ipynb [hunk 2]
```diff
@@ -1188,7 +1188,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Load the WFSS image into the appropriate model (hint: ImageModel)\n",
+    "# Load the WFSS image (wfss_rate_file[1]) into the appropriate model (hint: ImageModel)\n",
```
Comment (eteq) [inline pipeline_products_session/jwst-data-products-part2-live.ipynb:1191]:
```suggestion
    "# Load the WFSS image (`wfss_rate_file[1]`) into the appropriate model (hint: ImageModel)\n",
```

File: pipeline_products_session/jwst-data-products-part2-solutions.ipynb [hunk 3]
```diff
@@ -492,7 +492,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Load the WFSS image into the appropriate model (hint: ImageModel)\n"
+    "# Load the WFSS image (wfss_rate_file[1]) into the appropriate model (hint: ImageModel)\n"
```
Comment (eteq) [inline pipeline_products_session/jwst-data-products-part2-solutions.ipynb:495]:
```suggestion
    "# Load the WFSS image (`wfss_rate_file[1]`) into the appropriate model (hint: ImageModel)\n"
```

File: pipeline_products_session/jwst-data-products-part2-solutions.ipynb [hunk 4]
```diff
@@ -1240,7 +1240,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Load the WFSS image into the appropriate model (hint: ImageModel)\n",
+    "# Load the WFSS image (wfss_rate_file[1]) into the appropriate model (hint: ImageModel)\n",
```
Comment (eteq) [inline pipeline_products_session/jwst-data-products-part2-solutions.ipynb:1243]:
```suggestion
    "# Load the WFSS image (`wfss_rate_file[1]`) into the appropriate model (hint: ImageModel)\n",
```

**Review by eteq — APPROVED** [review]
Looks good, but some very minor suggestions!

**Review by aliciacanipe — COMMENTED** [review]
Thanks @eteq , looks good

**Pull request merged**